docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -3709,27 +3710,12 @@ Use `FAST` for [fast synchronization](../../Concepts/Node-Types.md#run-a-full-no
   `FAST` is also the default if connecting to Ethereum Mainnet by not specifying the [`--network`](#network) and
```

Comment:
would this be clearer -
  `FAST` is also the default if running besu on the default network (Ethereum Mainnet) by specifying neither network or genesis file